src/components/filter/criteriaBased/CriteriaBasedFilterEditionDialog.tsx

```diff
@@ -46,15 +46,15 @@ const formSchema = yup
     .required();
 
 export interface CriteriaBasedFilterEditionDialogProps {
-    id: string;
+    id: string | undefined | null;
     name: string;
     titleId: string;
     open: boolean;
     onClose: () => void;
     broadcastChannel: BroadcastChannel;
     getFilterById: (id: string) => Promise<any>;
-    selectionForCopy: SelectionCopy;
-    setSelelectionForCopy: (selection: SelectionCopy) => Dispatch<SetStateAction<SelectionCopy>>;
+    selectionForCopy: any;
+    setSelectionForCopy: (selection: any) => void;
```

Comment:
Why did you pass this to `any`??  
If there is a problem in gridexplore, either fix the type or add a todo, but don't pass to any just to disable alerts.

Reply:
Same has been done in previous typescript migration (ExpertFilterEditionDialog, ExplicitNamingFilterEditionDialog)
And we need to be homogenous between the different filter edition dialogs ...